event/producer_test.go

```diff
@@ -39,6 +39,7 @@ var (
 		Summary:         "",
 		ReleaseDate:     "",
 		Title:           "",
+		Topics:          []string{""},
```

Comment:
I have a feeling that you were trying to initialize a string array with no elements. If that is the case, then please look at the suggestion. Right now, you have initialized a string array with 1 element which is an empty string

The same is done for `Keywords` as well

```suggestion
		Topics:          []string{},
```